### PR TITLE
Th 64/view all mentoring sessions

### PR DESCRIPTION
**Pull request by khyechun**
## Context

JIRA ticket: https://teamhalpert.atlassian.net/jira/software/projects/TH/boards/1/backlog?selectedIssue=TH-64

## Checklist

- [ ] My branch is up-to-date with the `main` branch
- [ ] Unit and Integration test written
- [ ] Acceptance Criteria considered and implemented
- [ ] Automated tests are passing

### CODE REVIEW

File: src/features/Sessions/Sessions.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  // 2.Create Session will be in a separate folder : CreateSession
+  // 3.Session details will also be in a separate folder: Session Details
+  // a.Inside the folder, we will have two pages: SessionDetails and EditSessionDetails
+  const dispatch = useAppDispatch()
+  const { role } = useAppSelector(getAuth)
+  const {
+    menteeId,
+  } = useAppSelector(getBasicDetails)
+  const { options: assignedMenteeOptions, menteeIdToMenteeName } = useAssignedMenteesOptions()
+  const handleMenteeChange = (e: ChangeEvent<HTMLSelectElement>) => dispatch(setMentee({ menteeId: e.target.value, menteeName: menteeIdToMenteeName[e.target.value] }))
```
Comment (khyechun) [inline src/features/Sessions/Sessions.tsx:33]: this setMentee is for the form in create mentoring journey. Just create a local state for [mentee, setMentee]

File: src/features/Sessions/Sessions.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+  useEffect(() => {
+    if (assignedMenteeOptions.length > 0 && !menteeId.value) {
+      const firstMenteeId = assignedMenteeOptions[0].value
+      dispatch(setMentee({ menteeId: firstMenteeId, menteeName: menteeIdToMenteeName[firstMenteeId] }))
```
Comment (khyechun) [inline src/features/Sessions/Sessions.tsx:38]: Same here, dont need the dispatch. just use the setMentee from local state.

File: src/features/Sessions/SessionsTable/PendingSessionsTable.tsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+function PastSessionsTable() {
+  const { role } = useAppSelector(getAuth)
+  const TableComponent = role === 'Mentor' ? PastSessionsTableMentor : PastSessionsTableMentee
+  return (
+    <TableComponent />
+  )
+}
+
+function PastSessionsTableMentor() {
```
Comment (khyechun) [inline src/features/Sessions/SessionsTable/PendingSessionsTable.tsx:15]: Wrong name

File: src/features/Sessions/SessionsTable/PendingSessionsTable.tsx [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+            <Td />
+            <Td />
+          </Tr>
+        </Tbody>
+      </Table>
+    </TableContainer>
+  )
+}
+
+function PastSessionsTableMentee() {
```
Comment (khyechun) [inline src/features/Sessions/SessionsTable/PendingSessionsTable.tsx:44]: Wrong name

File: src/features/Sessions/Sessions.tsx [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+    }
+  }, [assignedMenteeOptions, dispatch, menteeId.value, menteeIdToMenteeName])
+
+  return (
+    <Container minHeight="100vh">
```
Comment (khyechun) [inline src/features/Sessions/Sessions.tsx:43]: isn't 100vh larger than the screen size?
you need to change this to calc(100vh - {value}px/rem)

File: src/features/Sessions/Calendar/Calendar.tsx [hunk 6]
```diff
@@ -0,0 +1,10 @@
+import Container from '../../../components/Container'
+
+function Calendar() {
+  return (
+    <Container backgroundColor="secondary.100" height="400px" minWidth="100vh">
```
Comment (khyechun) [inline src/features/Sessions/Calendar/Calendar.tsx:5]: Min width 100vh?

**Comment by lanceljr** [comment]
Resolved comments

**Pull request closed without merging**